asana/client.py

```diff
@@ -83,8 +83,16 @@ def request(self, method, path, **options):
                 response = getattr(self.session, method)(
                     url, auth=self.auth, **request_options)
                 if response.status_code in STATUS_MAP:
-                    raise STATUS_MAP[response.status_code](response)
-                elif response.status_code >= 500 and response.status_code < 600:
+                    # TODO: Change back to default behavior once 1.0 & 1.1 have
+                    # the same premium only response
+                    asana_error = STATUS_MAP[response.status_code](response)
+                    premium_only_str = "not available for free"
+                    if isinstance(asana_error, error.ForbiddenError) and (
+                            premium_only_str in str(asana_error)):
+                        raise error.PremiumOnlyError(response)
+                    raise asana_error
+                elif response.status_code >= 500 and (
+                        response.status_code < 600):
```

Comment:
This formatting is peculiar, and I think moving the opening paren to right after the `elif` would look cleaner.

Reply:
Yeah I've always had trouble figuring out the way that looks best for me for breaking up a if statement that's too long - looking over PEP8 again though, I think I'll go with what you're saying, which I think is this (https://www.python.org/dev/peps/pep-0008/#indentation)

```
if (this_is_one_thing
        and that_is_another_thing):
    do_something()
```